packages/core/index.js

```diff
@@ -3,12 +3,19 @@ const fs = require("fs");
 const path = require("path");
 const { getWeb3 } = require("@uma/common");
 
+function getDirectoryForVersion(version) {
+  const packageName = `@uma/core-${version.split(".").join("-")}`;
```

Comment:
Does this mean we would create a `core-1` directory containing older contracts...for example?

Reply:
I can add a comment on this because it's confusing. It basically resolves the directory containing the npm package `@uma/core-x-y-z`, where the version is `x.y.z`. So, this establishes a convention that we alias relevant core versions in the package.json as `@uma/core-x-y-z` as I've done in this PR with `1.1.0`. Does that make sense?

Reply:
ah very cool

Reply:
Ahhh, this is a great point. However, I don't think this index.js file will work at all in frontends right now due to the `path` import. I think the abis have to be pulled in manually regardless on the frontend (which you can do by importing the `.json` files directly). I think it makes sense to just leave this be and then figure out a separate method for the frontends down the road as you suggest. +1. Thanks for making this point.